src/commands/runtime/activation/list.js

```diff
@@ -214,7 +214,7 @@ ActivationList.flags = {
   // example usage:  aio runtime:activation:list --limit 10 --skip 2
   limit: flags.integer({
     char: 'l',
-    description: `only return LIMIT number of activations from the collection with a maximum LIMIT of ${ActivationList.limits.max} activations (default 30)`
+    description: `only return LIMIT number of activations (maximum ${ActivationList.limits.max}). Note that the maximum set for Adobe I/O Runtime is 50`
```

Comment:
Do you have to repeat this message on al the list commands?

mid the Max says 50 already, do you need the second Adobe part :)